js/src/wrappers/openai.ts

```diff
@@ -60,6 +56,108 @@ type PatchedOpenAIClient<T extends OpenAIType> = {
   };
 };
 
+function _combineChatCompletionChoices(
+  choices: OpenAI.ChatCompletionChunk.Choice[]
+): any {
+  const reversedChoices = choices.slice().reverse();
+  const message: { [key: string]: any } = {
+    role: "assistant",
+    content: "",
+  };
+  for (const c of reversedChoices) {
+    if (c.delta.role) {
+      message["role"] = c.delta.role;
+      break;
+    }
+  }
+  const toolCalls: {
+    [
+      key: number
+    ]: Partial<OpenAI.Chat.Completions.ChatCompletionChunk.Choice.Delta.ToolCall>[];
+  } = {};
+  for (const c of choices) {
+    if (c.delta.content) {
+      message.content += c.delta.content;
```

Comment:
Discussed offline, but may want to check this works even if content were a list (not unlikely some time in the future)